atomic-red-team/src/openbas_atomic_red_team.py

```diff
@@ -146,6 +146,7 @@ def _process_message(self) -> None:
                         platforms.sort()
                         payload = {
                             "payload_source": "COMMUNITY",
+                            "executable_arch": "x86_64",
```

Comment:
This is really what we want ? Because I don't have any architecture when I create a command line payload from the UI

Reply:
Yes, I'm making the changes on the Openbas side to take this into account. https://github.com/OpenBAS-Platform/openbas/pull/1885